service/component-samples/http-device-sample/src/main/java/org/fido/iot/sample/Device.java

```diff
@@ -213,79 +214,97 @@ protected MessagingService getMessagingService(Composite request) {
     client.run();
   }
 
+  boolean isRvBypassSet(Composite rvInformation) {
+    Composite rvItems = rvInformation.getAsComposite(Const.FIRST_KEY);
+    for (int i = 0; i < rvItems.size(); i++) {
+      Composite variable = rvItems.getAsComposite(i);
+      if (variable.getAsNumber(Const.FIRST_KEY).intValue() == Const.RV_BYPASS) {
+        return true;
+      }
+    }
+    return false;
+  }
+
   void doTransferOwnership(Composite credentials) {
 
     Composite rvi = credentials.getAsComposite(Const.DC_RENDEZVOUS_INFO);
-    List<String> to1Urls = RendezvousInfoDecoder.getHttpDirectives(rvi, Const.RV_DEV_ONLY);
 
     // the 'signed blob' contains the redirect to TO2
     AtomicReference<Composite> signedBlob = new AtomicReference<>();
     AtomicReference<Composite> wrappedCreds = new AtomicReference<>(credentials);
     AtomicBoolean isTo2Done = new AtomicBoolean(false);
 
-    To1ClientStorage to1Storage = new To1ClientStorage() {
+    rvBypass = isRvBypassSet(rvi);
+    if (!rvBypass) {
+      // if rvBypass flag is not set, continue with T01.
```

Comment:
Instead of this comment, put a logger statement that says "RvBypass flag not set, Starting TO1". Similar to else logger statement.

Reply:
Added logger statement.